packages/DevTools/src/signature-editor/const.ts

```diff
@@ -0,0 +1,215 @@
+import * as DG from 'datagrok-api/dg';
+
+//help with examples
+
+export const DEFAULT_CATEGORY = 'Misc';
+
+export enum FUNC_PROPS_FIELD {
+  NAME = 'name',
+  DESCRIPTION = 'description',
+  LANGUAGE = 'language',
+  HELP_URL = 'helpUrl',
+  REFERENCE = 'reference',
+  LOGIN = 'author?.login',
+  SAMPLE = 'sample',
+  ENVIRONMENT = 'environment',
+  TAGS = 'tags',
+  CONNECTION = "connection"
+}
+
+export const tooltipMessage = {
+  'caption': 'Custom field caption',
+  'postfix': 'Field postfix',
+  'units': 'Value unit name',
+  'editor': 'Editor',
+  'semType': 'Semantic type',
+  'columns': 'Numerical or categorical columns will be loaded',
+  'type': 'Column type',
+  'format': 'Datetime format',
+  'allowNulls': 'Validation of the missing values presence',
+  'action': 'For output parameters only',
+  'choices': 'List of choices for string parameter',
+  'suggestions': 'List of suggestions for string parameter',
+  'min': 'Minimum value',
+  'max': 'Maximum value',
+}
+
+export const obligatoryFuncProps = ['name', 'description', 'helpUrl', 'language'];
+
+export const functionPropsLabels = (key: FUNC_PROPS_FIELD) => {
+  switch (key) {
+    case FUNC_PROPS_FIELD.NAME: return 'Name';
+    case FUNC_PROPS_FIELD.CONNECTION: return 'Connection';
+    case FUNC_PROPS_FIELD.DESCRIPTION: return 'Description';
+    case FUNC_PROPS_FIELD.LOGIN: return 'Author';
+    case FUNC_PROPS_FIELD.HELP_URL: return 'Help URL';
+    case FUNC_PROPS_FIELD.LANGUAGE: return 'Language';
+    case FUNC_PROPS_FIELD.REFERENCE: return 'Reference';
+    case FUNC_PROPS_FIELD.SAMPLE: return 'Sample';
+    case FUNC_PROPS_FIELD.ENVIRONMENT: return 'Environment';
+    case FUNC_PROPS_FIELD.TAGS: return 'Tags';
+  }
+};
```

Comment:
instead of using `switch` you can try to make something like this:
`
export const FUNCTION_PROPS_LABELS: {[key: FUNC_PROPS_FIELD]: string} = {
  FUNC_PROPS_FIELD.NAME: 'Name',
  FUNC_PROPS_FIELD.CONNECTION: 'Connection'
  ...
};
`
so then you just need to write `FUNCTION_PROPS_LABELS[someVariable]` and you'll get the value just by accessing it by property name